contracts/root/RootChain.sol

```diff
@@ -131,4 +125,13 @@ contract RootChain is RootChainStorage, IRootChain {
         headerBlock.root = bytes32(dataList[3].toUintStrict());
         headerBlock.createdAt = now;
     }
+
+    // HOUSEKEEPING functions
+    function setNextHeaderBlock(uint256 _value) public onlyOwner {
+        _nextHeaderBlock = _value;
+    }
+
+    function setHeimdallId(string memory _heimdallId) public onlyOwner {
```

Comment:
how are we setting/updating `registry`?

Reply:
before it was in the constructor.

Reply:
Missed committing `RootChainProxy`. Check now.